integration_tests/cork_test.go

```diff
@@ -22,292 +20,456 @@ import (
 	"google.golang.org/grpc/credentials"
 )
 
-func (s *IntegrationTestSuite) TestCork() {
-	s.Run("Bring up chain, and submit a cork", func() {
-		s.T().Logf("checking that test cellar exists in the chain")
-		val := s.chain.validators[0]
-		s.Require().Eventuallyf(func() bool {
-			queryClient, err := val.GetQueryClient()
-			s.Require().NoError(err, "error getting query client")
-
-			res, err := queryClient.QueryCellarIDs(context.Background(), &corkTypes.QueryCellarIDsRequest{})
-			if err != nil {
-				return false
-			}
-			if res == nil {
-				return false
-			}
-			for _, c := range res.CellarIds {
-				if c == hardhatCellar.String() {
-					return true
-				}
-			}
-			return false
-		}, 30*time.Second, 2*time.Second, "hardhat cellar not found in chain")
-
-		s.T().Logf("wait for new vote period start")
-		s.Require().Eventuallyf(func() bool {
-			kb, err := val.keyring()
-			s.Require().NoError(err)
-			clientCtx, err := s.chain.clientContext("tcp://localhost:26657", &kb, "val", val.keyInfo.GetAddress())
-			s.Require().NoError(err)
+// Steward/Cork integration tests follow the same basic pattern:
+// 1. Query the cork module to ensure the relevant cellar ID is approved
+// 2. Wait for the next cork vote period to begin
+// 3. Construct and send the relevant cork request to each steward instance
+// 4. Wait for the voting period to end so the cork will be submitted to the gravity module as a ContractCallTx
+// 5. Query the gravity module to ensure the ContractCallTx was created
+// 6. Wait for the LogicCallSubmitted event to occur in the gravity contract, indicating that the orchestrators
+//	  relayed the ContractCallTx successfully.
+// 7. Wait for the presumed Cellar contract event indicating a successful decoding of the function call corked by Steward.
+
+const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
+const ONE_ADDRESS = "0x1111111111111111111111111111111111111111"
+
+// AaveV2Stablecoin rebalance test values used in multiple methods
+const T_AAVE_DAI = "0x6B175474E89094C44Da98b954EedeAC495271d0F"
+const T_AAVE_POOL = "0xE340B25fE32B1011616bb8EC495A4d503e322177"
+const T_AAVE_USDC = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
+
+// 1*10^18
+const T_AAVE_MIN_ASSETS_OUT = "1000000000000000000"
+
+func (s *IntegrationTestSuite) TestAaveV2Stablecoin() {
+	// s.Run("Submit rebalance request to MockAaveV2Stablecoin", func() {
```

Comment:
Both of the integration tests are fully commented out. Are they broken?

Reply:
Oops